test/OrchardCore.Tests/Modules/OrchardCore.Security/Extensions/ContentTypeOptionsBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace OrchardCore.Security.Extensions.Tests
+{
+    public class ContentTypeOptionsBuilderExtensionsTests
+    {
+        [Fact]
+        public void UseContentTypeOptionsWithoutOptionsShouldInjectDefaultHeader()
```

Comment:
Use such "should" verbiage in all test names.

Reply:
This isn't addressed. Please don't resolve convos that aren't addressed.

Reply:
Stange!! these test has been removed from awhile, I'm not sure why there are prenset here?!!

Reply:
The current test names are still doing something else. Why do you resolve this convo for a third time while not addressing my feedback? :) If it's not clear, I'm asking to use imperative language in test names, like you've done in this now removed test (and what most of the tests in `OrchardCore.Tests` do too).

Reply:
Test names should reflect what the test asserts. Examples of what I'm talking about:
- Instead of `AddSecurityHeaders_WithConfigureOptions` use `SecurityHeadersShouldReflectConfigureOptions` (I don't really see why `AddSecurityHeaders_WithBuildConfiguration` is needed too, BTW.)
- `ShouldAddContentSecurityPolicyHeader_WhenItsOptionsConfigured` -> `ContentSecurityPolicyHeaderShouldBeAdded`.

Also see the other, existing tests in the project for examples.

Reply:
And now the same in `SecurityHeadersApplicationBuilderExtensionsTests` ;).

Reply:
Please check now

Reply:
Looks good!